bindings/python/__init__.py

```diff
@@ -1,5 +1,8 @@
-from pkg_resources import resource_filename
+try:
+    from importlib.resources import files
+except ImportError:
+    from importlib_resources import files
 
 from .verovio import *
 
-verovio.setDefaultResourcePath(resource_filename('verovio', 'data'))
\ No newline at end of file
+verovio.setDefaultResourcePath(str(files("verovio") / "data"))
```

Comment:
Urghle... overloaded division operators as path separators. That's the smelliest part of Python IMO.